manifests/manage_binding.pp

```diff
@@ -33,22 +33,9 @@
         fail('https bindings require a valid ip_address')
       }
 
-      file { "inspect-${title}-certificate.ps1":
-        ensure  => file,
-        path    => "C:\\temp\\inspect-${name}.ps1",
-        content => template('iis/inspect-certificate-binding.ps1.erb'),
-      }
-
-      file { "create-${title}-certificate.ps1":
-        ensure  => file,
-        path    => "C:\\temp\\create-${name}.ps1",
-        content => template('iis/create-certificate-binding.ps1.erb'),
-      }
-
       exec { "Attach-Certificate-${title}":
-        command   => "C:\\temp\\create-${name}.ps1",
-        onlyif    => "C:\\temp\\inspect-${name}.ps1",
-        require   => [File["inspect-${title}-certificate.ps1"], File["create-${title}-certificate.ps1"]],
+        command   => template('iis/create-certificate-binding.ps1.erb'),
```

Comment:
Using template() rather than the file path to an executable is undocumented behaviour for both ```command``` and ```onlyif```. Also, where does the templated file end up?